helm/README.md

```diff
@@ -14,7 +14,7 @@ A Helm chart for installing Zaakafhandelcomponent
 
 | Repository | Name | Version |
 |------------|------|---------|
-| https://charts.bitnami.com/bitnami | solr | 9.1.6 |
+| https://charts.bitnami.com/bitnami | solr | 9.3.10 |
```

Comment:
This needs to be updated manually? It is even worthwhile to mention the version here? You can find that elsewhere no?

Reply:
We need to do `./gradlew build` on every helm chart change.

We can remove the dependent charts by supplying our own template. We're currently using the default one